system-int-test/test/cmr/system_int_test/ingest/bulk_update/bulk_update_test.clj

```diff
@@ -860,6 +864,36 @@
       ;; revision-id not changed, format not changed, it's identical to the original-concepts.
       (is (= new-concepts original-concepts)))) 
 
+(deftest bulk-update-update-all-tomb-stone-test
+  (let [coll1 (data2-core/ingest-umm-spec-collection "PROV1" (data-umm-c/collection {:EntryTitle "E1"
+                                                                                     :ShortName "S1"}))
+        _ (index/wait-until-indexed)
+        bulk-update-body {:concept-ids ["ALL"] 
+                          :update-type "FIND_AND_UPDATE"
+                          :update-field "SCIENCE_KEYWORDS"
+                          ;; Note: find-value is case-sensitive.
+                          :find-value {:Topic "aTmoSPHERE"}
+                          :update-value {:Category "EARTH SCIENCE"
+                                         :Topic "ATMOSPHERE"
+                                         :Term "AIR QUALITY"
+                                         :VariableLevel1 "EMISSIONS"}}]
+    ;; perform bulk update, verify that one collection is skipped because find-value is not found.
+    (let [response (ingest/bulk-update-collections "PROV1" bulk-update-body)
+          _ (index/wait-until-indexed)
+          collection-response (ingest/bulk-update-task-status "PROV1" (:task-id response))]
+      (is (= "COMPLETE" (:task-status collection-response)))
+      (is (= "Task completed with 1 SKIPPED out of 1 total collection update(s)." (:status-message collection-response))))
```

Comment:
nit: it would be better to have this update succeed without any special messages. `skipped` is not as nice as total success.

Reply:
I only need to distinguish the case when there is a collection to be processed or not.